.eslintrc

```diff
@@ -179,6 +179,14 @@
         "no-unused-expressions": 0,
         "react/no-find-dom-node": 0
       }
+    },
+    {
+      "files": [
+        "**/*.e2e.spec.js"
```

Comment:
Do we need "**/*.e2e.spec.jsx"?

Reply:
We only want to run it for the e2e tests currently. Open to doing this another way if there is a better way to do it. :)